content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -10,7 +10,7 @@ weight: 20
 <!-- overview -->
 {{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
-Kubernetes provides a [device plugin framework](https://git.k8s.io/design-proposals-archive/resource-management/device-plugin.md)
+Kubernetes provides a [device plugin framework](https://git.k8s.io/enhancements/keps/sig-node/3573-device-plugin/README.md)
```

Comment:
We shouldn't link to https://git.k8s.io/enhancements/keps/sig-node/3573-device-plugin/README.md with this link text, as that page doesn't mention the word _framework_ anywhere. This page itself is about device plugins.

I would omit the link here. We can link to https://kep.k8s.io/3573 in the **What's next** section at the foot of the page, if we want to.

Reply:
Hmm, yeah, makes sense.
Not sure what text would suit the link in the **What's next** section

Reply:
How about removing the hyperlink?